### PR TITLE
test: use latest conmon

### PR DESCRIPTION
**Pull request by haircommander**
#### What type of PR is this?

> /kind api-change
> /kind bug
/kind ci
> /kind cleanup
> /kind dependency-change
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake
> /kind other

#### What this PR does / why we need it:
which will allow us to catch regressions early

Signed-off-by: Peter Hunt <pehunt@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
none
```

### CODE REVIEW

File: scripts/versions [hunk 1]
```diff
@@ -4,7 +4,7 @@ set -euo pipefail
 # Versions to be used
 declare -A VERSIONS=(
     ["cni-plugins"]=v0.8.7
-    ["conmon"]=v2.0.26
+    ["conmon"]=master
```
Comment (saschagrunert) [inline scripts/versions:7]: This version will be used for the static binary bundle, too. So it will not work for that case.

Comment (haircommander) [reply scripts/versions]: alas

**Comment by haircommander** [comment]
latest conmon is broken :)

**Comment by haircommander** [comment]
https://github.com/containers/conmon/pull/246 should help

**Comment by TomSweeneyRedHat** [comment]
LGTM
assuming happy tests.

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Pull request merged**